proxy/src/project.rs

```diff
@@ -53,15 +53,17 @@ pub struct Project {
 }
 
 /// Construct a Project from its metadata and stats
-impl Project {
+impl<ST> From<(project::Project<ST>, coco::Stats)> for Project
+where
+    ST: Clone,
+{
     /// Create a `Project` given a `librad` defined [`project::Project`] and the [`coco::Stats`]
     /// for the repository.
-    pub fn from_project_stats<ST>(project: project::Project<ST>, stats: coco::Stats) -> Self
-    where
-        ST: Clone,
-    {
+    fn from(tup: (project::Project<ST>, coco::Stats)) -> Self {
```

Comment:
```suggestion
    fn from((project, stats): (project::Project<ST>, coco::Stats)) -> Self {
```
I often forget this, but we can pattern match in functions

Reply:
very nice